src/rdkafka.h

```diff
@@ -4880,14 +4943,153 @@ struct rd_kafka_group_list {
  *           group list.
  *
  * @sa Use rd_kafka_group_list_destroy() to release list memory.
+ *
+ * @deprecated Use rd_kafka_describe_consumer_groups() instead.
  */
 RD_EXPORT
+RD_DEPRECATED
 rd_kafka_resp_err_t
 rd_kafka_list_groups(rd_kafka_t *rk,
                      const char *group,
                      const struct rd_kafka_group_list **grplistp,
                      int timeout_ms);
 
+/**
+ * @brief Creates an options struct for rd_kafka_list_consumer_groups.
+ *
+ * \p request_timeout the request timeout.
+ * \p states An array with the states to query, NULL to query for all the
+ * states. \p states_cnt Size of the state array or 0 if NULL.
+ *
+ * @returns A pointer to the newly created
+ * rd_kafka_list_consumer_groups_options_t,
+ * or NULL if options parameters are invalid.
+ *
+ * @sa Use rd_kafka_list_consumer_groups_options_destroy() to release it.
+ */
+RD_EXPORT
+rd_kafka_list_consumer_groups_options_t *
+rd_kafka_list_consumer_groups_options_new(
+    int request_timeout,
```

Comment:
This is already on the standard AdminOptions, isn't it?

Reply:
(and timeouts should always include the unit, e.g., request_timeout_ms)

Reply:
dito